src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,52 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import renderer from "react-test-renderer";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults", () => {
+  let validProps = {
+    mockResults: [
+      "https://testimage1",
+      "https://testimage2",
+      "https://testimage3",
+    ],
+    mockIsLoading: false,
+  };
+
+  it("renders SearchResults correctly", () => {
+    const rendered = renderer.create(
+      <SearchResults 
+        results={validProps.mockResults} 
+        isLoading={validProps.mockIsLoading} 
+      />
+    );
+
+    expect(rendered).toMatchSnapshot();
+  });
+
+  it("renders correct number of results", () => {
+    render(
+      <SearchResults 
+        results={validProps.mockResults} 
+        isLoading={validProps.mockIsLoading} 
+      />
+    );
+
+    expect(screen.getByAltText("https://testimage1")).toBeInTheDocument();
+    expect(screen.getByAltText("https://testimage2")).toBeInTheDocument();
+    expect(screen.getByAltText("https://testimage3")).toBeInTheDocument();
```

Comment:
This does work, but does it validate whether or not a 4th image was displayed? You add another set to do this and find all occurrences of images like:
```
    const images = screen.getAllByRole('img')
    expect(images).toHaveLength(validProps.mockResults.length)
```
so you can be sure there are no others being displayed somehow